pkg/ingestor/parser/csaf/parser_csaf.go

```diff
@@ -88,7 +88,7 @@ func (c *csafParser) GetIdentities(ctx context.Context) []common.TrustInformatio
 }
 
 func (c *csafParser) GetIdentifiers(ctx context.Context) (*common.IdentifierStrings, error) {
-	return nil, fmt.Errorf("not yet implemented")
+	return c.identifierStrings, nil
```

Comment:
need to append like the other parsers: https://github.com/guacsec/guac/blob/f9b507f56e6098f4d5ab13c110a99895b9e70083/pkg/ingestor/parser/spdx/parse_spdx.go#L117

Reply:
You're right, I have got it appending the product id’s to `PurlStrings`.